basket/src/App.tsx

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import { useState } from 'react';
+import { My } from './components/My';
+
+const SampleSession = {
```

Comment:
자바스크립트에서 변수명은 camelCase로 작성하기!

```suggestion
const sampleSession = {
```